account_invoice_blocking/models/account_invoice.py

```diff
@@ -75,11 +74,12 @@ def _compute_move_blocked(self):
             ) if move_lines else False
 
     blocked = fields.Boolean(
-        'No Follow-up',
```

Comment:
You removed the string here but the string to be removed was the one of field `draft_blocked`.
Moreover you didn't add the corresponding string in the view.